staging/src/k8s.io/client-go/tools/pager/pager.go

```diff
@@ -123,7 +123,7 @@ func (p *ListPager) List(ctx context.Context, options metav1.ListOptions) (runti
 			list.ResourceVersion = m.GetResourceVersion()
 			list.SelfLink = m.GetSelfLink()
 		}
-		if err := meta.EachListItem(obj, func(obj runtime.Object) error {
+		if err := meta.EachListItemWithAlloc(obj, func(obj runtime.Object) error {
```

Comment:
same change on line 161 below?

Reply:
Thanks for the reminder, it has been fixed